packages/react/src/components/Popup/Popup.tsx

```diff
@@ -59,17 +57,15 @@ export interface PopupSlotClassNames {
 export interface PopupProps
   extends StyledComponentProps<PopupProps>,
     ChildrenComponentProps,
-    ContentComponentProps<ShorthandValue> {
+    ContentComponentProps<ShorthandValue>,
+    PositionCommonProps {
```

Comment:
nit: we may make it shorter and rename to `PositioningProps`

Reply:
done, let's try to get all comments on the initial review, this interface was there from beginning. of course I understand it might have come to your attention on the 2nd round. thanks for reviewing 👍